src/pages/in/pricing.js

```
@@ -1,9 +1,1248 @@
+   * @returns
+   */
+  const getPlanInfoByFeature = (plan, feature) => {
+    if (!plan && !feature) {
+      return;
+    }
+
+    if (Object.hasOwn(feature, "yearly")) {
+      return plan[isYearPlan ? feature.yearly : feature.monthly];
+    } else if (feature.specificContent && !isFreePlan(plan)) {
+      return (
+        <span dangerouslySetInnerHTML={{ __html: feature.specificContent }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)